Libplanet/Net/Swarm.BlockCandidate.cs

```diff
@@ -0,0 +1,406 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Libplanet.Blockchain;
+using Libplanet.Blocks;
+
+namespace Libplanet.Net
+{
+    public partial class Swarm<T>
+    {
+        private async Task BlockCandidateConsume(
```

Comment:
It might be trivial, but IMHO it should be `ConsumeBlockCandidates`.

Reply:
https://github.com/planetarium/libplanet/pull/1699/commits/1c86d3717bdea923777f27354c06907575439caa fixed it!